driver-core/src/test/resources/retryable-writes/bulkWrite-serverErrors.json

```diff
@@ -13,9 +13,6 @@
   "tests": [
     {
       "description": "BulkWrite succeeds after PrimarySteppedDown",
-      "clientOptions": {
```

Comment:
Are these going to be pushed downstream?

Reply:
They are already there.  They are part of the commit that is references in the DRIVERS ticket